src/main/java/com/commercetools/sync/categories/CommonTypesDiff.java

```diff
@@ -0,0 +1,86 @@
+package com.commercetools.sync.categories;
+
+
+import io.sphere.sdk.categories.Category;
+import io.sphere.sdk.commands.UpdateAction;
+import io.sphere.sdk.models.LocalizedString;
+import io.sphere.sdk.models.Reference;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.Objects;
+import java.util.Optional;
+
+public class CommonTypesDiff {
+
+    /**
+     * Compares two {@link LocalizedString} and returns a supplied {@link UpdateAction < Category >} as a result in an
+     * {@link Optional}. If no update action is needed, for example in case where both the {@link LocalizedString}
+     * have the same fields and values, an empty {@link Optional} is returned.
+     *
+     * @param existingLocalisedString the localised string which should be updated.
+     * @param newLocalisedString      the localised string with the new information.
+     * @param updateAction            the update action to return in the optional.
+     * @return A filled optional with the update action or an empty optional if the localised string values are identical.
+     */
+    @Nonnull
+    static Optional<UpdateAction<Category>> buildUpdateActionForLocalizedStrings(
+            @Nullable final LocalizedString existingLocalisedString,
+            @Nullable final LocalizedString newLocalisedString,
+            @Nonnull final UpdateAction<Category> updateAction) {
+        return buildUpdateActionForObjects(existingLocalisedString, newLocalisedString, updateAction);
```

Comment:
Why do you have these extra methods and you don't use only `buildUpdateActionForObjects`?